docs/source/json-api/index.rst

```diff
@@ -1075,7 +1075,9 @@ off an initial "loading" indicator::
 To keep the stream alive, you'll occasionally see messages like this,
 which can be safely ignored::
```

Comment:
I would qualify under which circumstances you can ignore them, namely when you don't care about ledger offsets.